source/test/efficient_mode_test/EfficientSymbiont.test.cc

```diff
@@ -385,92 +393,92 @@ TEST_CASE("EfficientSymbiont's Process called from Host when mutation rate and s
+            } else if (host3->HasSym()) {
+                new_infected = host3;
+            } else if(host4->HasSym()) {
+                new_infected = host4;
             }
             REQUIRE(new_infected != nullptr);
-            REQUIRE(new_infected->HasSym()); 
+            REQUIRE(new_infected->HasSym());
             REQUIRE(new_infected->GetSymbionts()[0]->GetEfficiency() == efficiency);
         }
-
     }
 
-    WHEN("The horizontal mutation rate and size are not zero and an EfficientSymbiont is added to a Host and about to reproduce horizontally and Host's Process is called") {
+    WHEN("The horizontal mutation rate and size are not zero and an EfficientSymbiont is added to a Host and about to reproduce horizontally and Host'symbiont Process is called") {
         config.HORIZ_MUTATION_SIZE(0.002);
         config.HORIZ_MUTATION_RATE(1.0);
-        EfficientHost * h = new EfficientHost(random, &w, &config, host_interaction_val, {}, {}, std::set<int>(), host_points);
-        EfficientHost * h2 = new EfficientHost(random, &w, &config, host_interaction_val, {}, {}, std::set<int>(), host_points);
-        EfficientHost * h3 = new EfficientHost(random, &w, &config, host_interaction_val, {}, {}, std::set<int>(), host_points);
-        EfficientHost * h4 = new EfficientHost(random, &w, &config, host_interaction_val, {}, {}, std::set<int>(), host_points);
+        emp::Ptr<EfficientHost> host = emp::NewPtr<EfficientHost>(random, &w, &config, host_interaction_val);
+        emp::Ptr<EfficientHost> host2 = emp::NewPtr<EfficientHost>(random, &w, &config, host_interaction_val);
+        emp::Ptr<EfficientHost> host3 = emp::NewPtr<EfficientHost>(random, &w, &config, host_interaction_val);
+        emp::Ptr<EfficientHost> host4 = emp::NewPtr<EfficientHost>(random, &w, &config, host_interaction_val);
 
-        EfficientSymbiont * s = new EfficientSymbiont(random, world, &config, int_val, points, efficiency);
-        h->AddSymbiont(s);
-        w.AddOrgAt(h, 0);
-        w.AddOrgAt(h2, 1);
-        w.AddOrgAt(h3, 2);
-        w.AddOrgAt(h4, 3);
+        emp::Ptr<EfficientSymbiont> symbiont = emp::NewPtr<EfficientSymbiont>(random, world, &config, int_val, points, efficiency);
+        host->AddSymbiont(symbiont);
+        w.AddOrgAt(host, 0);
+        w.AddOrgAt(host2, 1);
+        w.AddOrgAt(host3, 2);
+        w.AddOrgAt(host4, 3);
 
-        h->Process(0);
+        host->Process(0);
 
         THEN("EfficientSymbiont reproduces and offspring with a different efficiency value goes into neighboring Host") {
-            EfficientHost * new_infected = nullptr;
-            if(h2->HasSym()) {
-                new_infected = h2;
-            } else if (h3->HasSym()) {
-                new_infected = h3;
-            } else if(h4->HasSym()) {
-                new_infected = h4;
+            emp::Ptr<EfficientHost> new_infected = nullptr;
+            if(host2->HasSym()) {
+                new_infected = host2;
```

Comment:
Could make a list of the non-infected and check that they don't have symbionts to make the codecov happy